recipe/meta.yaml

```diff
@@ -14,42 +25,95 @@ source:
     - patches/0004-add-missing-osx-workaround.patch
 
 build:
-  number: 0
-  run_exports:
-    - {{ pin_subpackage("abseil-cpp", max_pin="x.x") }}
+  number: 1
 
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cmake
-    - ninja
+outputs:
+  {% if shared_libs == "ON" %}
+  - name: libabseil_shared
```

Comment:
Do we need this output name? I would assume that `libabseil` (the alias below) is shared.

Reply:
I've tried to keep the package doing what it used to be doing, i.e. shared on unix, static on windows. That's just to ease transition from the current state of affairs though.

I'm happy to get rid of one of that though, would make it clearer that windows has no shared libs, for example (and we could still keep the current behaviour in `abseil-cpp`).

Reply:
This is done now